openapi/multi-yaml/paths/account@{account_id}@payout_holds@{payout_hold_id}.yaml

```diff
@@ -0,0 +1,55 @@
+get:
+  summary: Get a Payout Hold
+  description: Retrieve a specific payout hold by ID.
+  operationId: GetPayoutHold
+  tags:
+    - Payout Holds
+  parameters:
+    - $ref: ../components/parameters/account-id-path.yaml
```

Comment:
Same here - see my earlier question about why the account_id is needed in the path.